### ISSUE TITLE
OnePlus 3 after Oreo update isn't recognized

### ISSUE THREAD

**Issue by TamasBarta** [M0Rf30/android-udev-rules#131]
After upgrading my OnePlus 3 to Oreo, the `lsusb` command returns `2a70:4ee7`, so I don't have permission to use it. I manually created `/etc/udev/rules.d/51-android.rules`:

```
SUBSYSTEM=="usb",ATTR{idVendor}=="2a70", MODE="0660", GROUP="adbusers"
SUBSYSTEM=="usb",ATTR{idVendor}=="2a70",ATTR{idProduct}=="4ee7",SYMLINK+="android_adb"
SUBSYSTEM=="usb",ATTR{idVendor}=="2a70",ATTR{idProduct}=="4ee7",SYMLINK+="android_fastboot"
```

Then I ran `sudo udevadm control --reload-rules` as mentioned in the [Wiki](https://wiki.archlinux.org/index.php/android#Connect_device).

This solved the problem, but I believe it needs more action, though I'm not sure yet what. This Oreo update is pulled back by OnePlus, this is not accessible anymore. I also started a [thread about it](https://forums.oneplus.net/threads/usb-vendor-and-product-id-changed-after-oreo-update.707958/) on their forums, so we can find out if it is intentional, or my device is in a broken state regarding these IDs.

**Comment by M0Rf30** [comment]
fixed by #130 and #129

**Comment by TamasBarta** [comment]
Awesome! Thanks!
